src/multibody/geometry.hxx

```diff
@@ -47,6 +47,20 @@ namespace se3
     return idx;
   }
 
+  inline GeomIndex GeometryModel::addGeometryObject(const JointIndex parent,                                                    
+                                                    const SE3 & placement,
+                                                    const std::string & geom_name,
+                                                    const std::string & mesh_path) throw(std::invalid_argument)
+  {
+
+    Index idx = (Index) (ngeoms ++);
+    const boost::shared_ptr<fcl::CollisionGeometry> geometry(new fcl::CollisionGeometry());
```

Comment:
Nota: WITH_HPP_FCL  CollisionGeometry is a pure virtual Class. Travis fails because of it